beater/test_approved_es_documents/TestPublishIntegrationProfileCPUProfile.approved.json

```diff
@@ -0,0 +1,9500 @@
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/dial.go",
+                        "function": "net.(*sysDialer).dialSingle",
+                        "id": "c873b23502e2a074",
+                        "line": 578
+                    },
+                    {
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/dial.go",
+                        "function": "net.(*sysDialer).dialSerial",
+                        "id": "988ba8ff8d1f0012",
+                        "line": 546
+                    },
+                    {
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/dial.go",
+                        "function": "net.(*Dialer).DialContext",
+                        "id": "578e314c9c012074",
+                        "line": 424
+                    },
+                    {
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/http/transport.go",
+                        "function": "net/http.(*Transport).dial",
+                        "id": "ff6df33ebd67a383",
+                        "line": 1043
+                    },
+                    {
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/http/transport.go",
+                        "function": "net/http.(*Transport).dialConn",
+                        "id": "0a7feedb8232ad47",
+                        "line": 1470
+                    },
+                    {
+                        "filename": "/home/andrew/tools/go/1.13.0/src/net/http/transport.go",
+                        "function": "net/http.(*Transport).dialConnFor",
+                        "id": "8e7960e2a57f4ad2",
+                        "line": 1300
+                    }
+                ],
+                "top": {
+                    "filename": "/home/andrew/tools/go/1.13.0/src/runtime/sys_linux_amd64.s",
+                    "function": "runtime.epollctl",
+                    "id": "32cd0e3183bb82bc",
+                    "line": 660
+                }
+            }
+        },
+        {
+            "@metadata": {
+                "beat": "apm-test",
+                "pipeline": "apm",
+                "type": "_doc",
+                "version": "8.0.0"
+            },
+            "@timestamp": "2019-11-22T10:30:36.305Z",
+            "ecs": {
+                "version": "1.2.0"
+            },
+            "observer": {
+                "ephemeral_id": "00000000-0000-0000-0000-000000000000",
+                "hostname": "",
+                "id": "fbba762a-14dd-412c-b7e9-b79f903eb492",
+                "type": "test-apm-server",
+                "version": "8.0.0",
+                "version_major": 8
+            },
+            "processor": {
+                "event": "profile",
+                "name": "profile"
+            },
+            "profile": {
+                "cpu.ns": 20000000,
+                "duration": 5100778105,
+                "id": "d58847c89c275eba",
```

Comment:
Shouldn't the `id` of a profile be unique, and the `timestamp` usually differ per profile? Is it possible to extract 2 or 3 unique profiles per example?

Reply:
A profile is just a collection of samples, and each document corresponds to one sample. I added the `profile.id` field so that we could group all samples by the profile that they came from.

I've since removed this field for a few reasons:
 - It's not needed right now
 - We can potentially use the `@timestamp` field instead (all samples from a profile will have the same timestamp value)
 - The field won't be very meaningful if we start performing additional aggregation of samples in the server